bedrock/firefox/templates/firefox/all/product.html

```diff
@@ -0,0 +1,74 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{# Choose product #}
+
+<h2 class="c-step-name">
+  1. {{ ftl('firefox-all-browser') }}
+  {% if product %}
+    <span class="c-step-choice">{{ product and product.name }}</span> <a href="{{ url('firefox.all') }}" class="c-step-icon"><img alt="{{ ftl('firefox-all-change-browser') }}" src="{{ static('protocol/img/icons/close.svg') }}" width="30" height="30"></a>
+  {% else %}
+    <img alt="{{ ftl('firefox-all-down-arrow') }}" class="c-step-icon" src="{{ static('protocol/img/icons/arrow-down.svg') }}" width="30" height="30">
+  {% endif %}
+
+  {% if not product %}
+  <a id="icon-browser-help" href="#browser-help" class="c-button-help" title="{{ ftl('firefox-all-learn-about-firefox') }}">
+    {{ ftl('firefox-all-get-help') }}
+  </a>
+  {% endif %}
+</h2>
+
+{% if not product %}
+  <div class="c-step-contents">
+    <h3 class="c-product-title"><img alt="" role="presentation" src="{{ static('protocol/img/icons/desktop.svg') }}"> Desktop</h3>
+    <ul class="c-product-list qa-desktop-list">
+      <li class="release"><a href="{{ url('firefox.all.platforms', product_slug='desktop-release') }}">Firefox</a> <strong>(Recommended)</strong></li>
```

Comment:
**issue:** should "(Recommended)" be translated?

Reply:
✅